main/install/configuration.dist.php

```diff
@@ -221,6 +221,12 @@
 //);
 // Course log - Default columns to hide
 //$_configuration['course_log_hide_columns'] = ['columns' => [1, 9]];
+// Course log - User extra fields to show as columns for default
+// For example, to show as a default column the user extra field with id 15 and 37 on site 1:
+// [1 => [15, 37]];
```

Comment:
Site 1 ? Site 3? What's that?

The configuration should be more clear for example:

```
$_configuration['course_log_default_extra_fields'] = [
    'extra_fields' => [
        'variable_test', 
        'variable_demo'
    ]
];
```
If the configuration is an array it should start with key of type string, in the example above the key is 'extra_fields'.

Or in this example : 

`$_configuration['course_log_hide_columns'] = ['columns' => [1, 9]];`

The key is "columns" otherwise there are conflicts with the Access url.

Reply:
Change the configuration to :

`$_configuration['course_log_default_extra_fields'] = ['extra_fields' => [66, 67]];`

Reply:
Use extra field variables instead of extra field ids, it will be more easy to edit later.

Reply:
Fixed!: https://github.com/chamilo/chamilo-lms/pull/3909/commits/242c5e90b457b93d99fd7c65eb70d5f555de90ae